### PR TITLE
adding more details about comprehension expectations

### PR DESCRIPTION
**Pull request by bundacia**

### CODE REVIEW

File: Phases/program-faqs.md [hunk 1]
```diff
@@ -32,6 +32,24 @@ No, not exactly. If a learner moves into phases 2, 3, or 4 early, the weeks they
 
 Learners who complete, understand, and clearly explain 80% or more of the content covered in their phase interview will be accepted into the phase for which they’re interviewing.
 
+#### What am I expected to be able to explain in a comprehension interview?
+
+In your phase interview the interviewer is going to try and get a sense of your comprehension of the code you submitted, so they will be asking you to explain parts of it. You won't be expected to explain abstract technical ideas that are unrelated to your code, but you *will* be expected to be able to explain the code you wrote and how it works on a basic level.
+
+Here's a non-technical analogy.
```
Comment (jaredatron) [inline Phases/program-faqs.md]: Would a technical analogy work better?

Lets say you're asked to write an `add` function. So you write:

```js
const add = function(a,b){ return a + b }
```
You will be asked to explain how each individual parts of this code work. So the interviewer might ask:

- What does the `const add =` part of this code do?
- Why did you name your arguments `a` and `b`?
- if `a` is `12` and `b` is `3`, what will `a + b` result in?
- What will the `add` function return when called like this `add(10, 5)`
- What will the `add` function return when called like this `add(7)`
- What will the `add` function return when called like this `add()`

**Review by jaredatron — APPROVED** [review]

File: Phases/program-faqs.md (hunk 1 above)
Comment (bundacia) [reply Phases/program-faqs.md]: yeah, I like

**Comment by needdra** [comment]
excellent. i'm going to merge this now.

File: Phases/Interviews/README.md [hunk 2]
```diff
@@ -13,7 +13,23 @@ Following submission, the evaluating SEP \(also referred to as the "evaluator"\)
...
 
-The technical comprehension interview is a 1-on-1 conversation with an SEP designed to evaluate your understanding of the solutions you came up with on the coding challenge. You'll meet with an SEP via video chat for about 30-45 minutes to dig in to various parts of the submitted code.
+The technical comprehension interview is a 1-on-1 conversation with an SEP designed to evaluate your understanding of the solutions you came up with on the coding challenge. You'll meet with an SEP via video chat for about 30-45 minutes to dig in to various parts of the submitted code. You won't be expected to explain abstract technical ideas that are unrelated to your code, but you *will* be expected to be able to explain the code you wrote and how it works on a basic level. 
+
+Here's an example. Lets say you're asked to write an `add` function. So you write:
```
Comment (jaredatron) [inline Phases/Interviews/README.md:18]: 👍

**Pull request merged**